src/server/components/charts-engine/components/storage/united-storage/provider.ts

```diff
@@ -40,9 +40,12 @@ axiosRetry(axios, {
     },
 });
 
+export const TEN_SECONDS = 10000;
```

Comment:
Maybe we can use these constants everywhere https://github.com/datalens-tech/datalens-ui/blob/58c19ee85dac61dfe1e690ab28e0e4452dda7f13/src/shared/constants/timeout.ts#L3?

Reply:
Agree

Reply:
done